.nvmrc

```diff
@@ -0,0 +1 @@
+v15.14.0
```

Comment:
odd Node versions are typically not meant to be used by end users but rather library authors testing future versions - not sure the nature of the tool, perhaps you are intentionally doing this.

14 is the current safe bet, with 16 ready in October

> https://nodejs.org/en/about/releases/

Reply:
This is more of a developer convenience in that having an `.nvmrc` helps keep Node versions consistent for contributors. Do you see a practical problem with this? I don't think it should affect end users since this file is not published in the npm registry.

Reply:
agree that having an nvmrc of any value is better over nothing - any consistency and automated config we can introduce into our lives is good.



> Do you see a practical problem with this?

I don't see practical problems in a sense that this is broken; only that it's non-standard from my experience. 

Even Node's front page ignores 15 because its a bit in no-man's land between the current long term support(14) and the current cutting edge (16)

![image](https://user-images.githubusercontent.com/298435/123557727-31f27800-d758-11eb-9537-d90a6ddc901c.png)

Reply:
New commit changes .nvmrc to point to v14.17.1.